contrib/comfort_models.py

```diff
@@ -171,67 +215,86 @@ def comfPMV(ta, tr, vel, rh, met, clo, wme):
 
     ts = 0.303 * math.exp(-0.036 * m) + 0.028
     pmv = ts * (mw - hl1 - hl2 - hl3 - hl4 - hl5 - hl6)
-    ppd = 100.0 - 95.0 * math.exp(-0.03353 * pow(pmv, 4.0)
-        - 0.2179 * pow(pmv, 2.0))
+    ppd = 100.0 - 95.0 * math.exp(-0.03353 * pow(pmv, 4.0) - 0.2179 * pow(pmv, 2.0))
 
-    r = []
-    r.append(pmv)
-    r.append(ppd)
+    # todo I would recoommend to return json instead
 
-    return r
+    return [pmv, ppd]
 
 
-def comfPierceSET(ta, tr, vel, rh, met, clo, wme):
-    """
-    Function to find the saturation vapor pressure, used frequently
-    throughtout the comfPierceSET function.
+def comfPierceSET(ta, tr, vel, rh, met, clo, wme=0):
+    """ SET calculation using SI units
+
+    The function returns a list containing the following values listed in order as they appear below:
+    - pmv : Predicted mean vote
+    - ppd : Percent predicted dissatisfied [%]
+    - ta_adj: Air temperature adjusted for air speed [C]
+    - cooling_effect : The difference between the air temperature and adjusted air temperature [C]
+    - set: The Standard Effective Temperature [C] (see below)
+
+    :param ta: dry bulb air temperature, [C]
+    :type  ta: float
+    :param tr: mean radiant temperature, [C]
+    :type  tr: float
+    :param vel: air velocity, [m/s]
+    :type  vel: float
+    :param rh: relative humidity, [%]
+    :type  rh: float
+    :param met: metabolic rate, [met]
+    :type  met: float
+    :param clo: clothing insulation, [clo]
+    :type  clo: float
+    :param wme: external work, [met] default 0
+    :type  wme: float
+
+    :return: list containing estimated parameters [PMV, PPD, Ta_adj, cooling_effect, SET]
+    :rtype: list
+
     """
-    res = None
 
-    def findSaturatedVaporPressureTorr(T):
-        # calculates Saturated Vapor Pressure (Torr) at Temperature T  (C)
-        return math.exp(18.6686 - 4030.183 / (T + 235.0))
+    def findSaturatedVaporPressureTorr(t):
+        """ Estimate the saturation vapor pressure in [torr]
+
+        :param t: air temperature, [C]
+        :type  t: float
 
-    # Key initial variables.
-    VaporPressure = (rh * findSaturatedVaporPressureTorr(ta)) / 100
+        :return: saturated vapour pressure, [torr]
+        :rtype: float
+        """
+        return math.exp(18.6686 - 4030.183 / (t + 235.0))
+
+    # Initial variables as defined in the ASHRAE 55-2017
+    VaporPressure = rh * findSaturatedVaporPressureTorr(ta) / 100
     AirVelocity = max(vel, 0.1)
     KCLO = 0.25
     BODYWEIGHT = 69.9
     BODYSURFACEAREA = 1.8258
     METFACTOR = 58.2
-    SBC = 0.000000056697  # Stefan-Boltzmann constant (W/m2K4)
```

Comment:
Seems like a helpful comment, maybe keep it?

Reply:
reverted my change, however, there are many lines that are not commented, hence, despite the fact that this comment is very useful, it would be hard for a user to follow the code line.